docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -266,38 +111,55 @@ spec:
               spec:
                 type: LoadBalancer
       manilaScheduler:
-        replicas: 3
+        replicas: 3 <2>
       manilaShares:
         cephfs:
-          replicas: 1
+          replicas: 1 <3>
           customServiceConfig: |
             [DEFAULT]
             enabled_share_backends = tripleo_ceph
             host = hostgroup
             [cephfs]
             driver_handles_share_servers=False
-            share_backend_name=cephfs
+            share_backend_name=cephfs <4>
             share_driver=manila.share.drivers.cephfs.driver.CephFSDriver
             cephfs_conf_path=/etc/ceph/ceph.conf
             cephfs_auth_id=openstack
             cephfs_cluster_name=ceph
             cephfs_volume_mode=0755
             cephfs_protocol_helper_type=CEPHFS
-          networkAttachments:
+          networkAttachments: <5>
               - storage
+      extraMounts: <6>
+      - name: v1
+        region: r1
+        extraVol:
+          - propagation:
+            - ManilaShare
+          extraVolType: Ceph
+          volumes:
+          - name: ceph
+            secret:
+              secretName: ceph-conf-files
+          mounts:
+          - name: ceph
+            mountPath: "/etc/ceph"
+            readOnly: true
 __EOF__
```

Comment:
@fmount I added the "extraMounts" content here. Does it look accurate?

Reply:
+1 on the extraMounts section, looks good, thank you!